_posts/2018-03-30-Introducing-Bazel-Cquery.md

```diff
@@ -17,22 +17,20 @@ The query commands also support a set of flag-based options that govern query be
-| Functions            | siblings, buildfiles, tests                                                       | config                                 | allpaths, attrs, dep, filter, kind, labels, loadfiles, rdeps, somepath, visible |
-| Output Formats       | build, label,  label_kind, minrank, maxrank, location, package, graph, xml, proto | label_and_configuration, transitions** |                                                                                 |
-| Options              | query options                                                                     | cquery options, build options          | common query options                                                            |
+|                      | `query`                                                                    |`cquery`                              | both                                                                            |
+|----------------------|----------------------------------------------------------------------------|--------------------------------------|---------------------------------------------------------------------------------|
+| Performance          | faster, less acurate                                                       | slower, accurate                     |                                                                                 |
+| Functions            | siblings, buildfiles, tests                                                | config                               | allpaths, attrs, dep, filter, kind, labels, loadfiles, rdeps, somepath, visible |
+| Output Formats       | build, label,  label_kind, minrank, maxrank, location, package, graph, xml | label_and_configuration, transitions | proto                                                                           |
+| Options              | query options                                                              | cquery options, build options        | common query options                                                            |
 
 So, if your priority is speed and over-approximation of results isn’t a problem, `query` is your engine. If your priority is results that match a specific bazel invocation’s flags and fancy output formats aren’t too important to you, then `cquery` is the better choice.
 
-\* Bazel also supports the ‘Sky Query’ engine which automatically kicks in with a specific set of options used with ‘query’. It supports a few extra functions and in some circumstances may be faster and less memory-intensive than ‘query’
-
-\*\* proto output format coming soon!
+\* Bazel also supports the `Sky Query` engine which automatically kicks in with a specific set of options used with `query`. It supports a few extra functions and in some circumstances may be faster and less memory-intensive than `query`
```

Comment:
What is `\*` supposed to mean?

Reply:
Escaping `*` for the HTML conversion

Reply:
Interesting, I don't see `\*` anywhere else in the post. Same question as @jmmv

Reply:
Ok, my bad, it's here: `We’re excited to announce a second* command`